### PR TITLE
Feature/add panel

### PR DESCRIPTION
**Pull request by skriyaz95**

### CODE REVIEW

File: src/components/explore/ListTemplate.vue [hunk 1]
```diff
@@ -22,16 +22,16 @@
         <v-list>
           <v-list-item v-for="(item, index) in itemsSorted" :key="index">
             <v-list-item-icon>
-              <v-icon>{{ icon }}</v-icon>
+              <v-icon :class="{'red--text': !item[fieldValidLabel]}">{{ icon }}</v-icon>
```
Comment (cmonnom) [inline src/components/explore/ListTemplate.vue]: Since this is used more than once, let's move it to a method returning the class name instead.

Comment (cmonnom) [reply src/components/explore/ListTemplate.vue]: Also, fieldValidLabel is a prop never set it seems. It's always 'valid'. So maybe we don't need it?

File: src/components/home/PanelResult.vue [hunk 2]
```diff
@@ -327,6 +333,23 @@ export default Vue.extend({
       }
       return 0
     },
+    formatGeneInfo(gene:Gene, symbolToSymbolMatchSet: Set<string>, synonymToSynonymMatchSet: Set<string>,
```
Comment (cmonnom) [inline src/components/home/PanelResult.vue:336]: I'm guessing this one is a work in progress. It's fine for now.

File: src/components/manage/BuildExploreInstitutions.vue [hunk 3]
```diff
@@ -138,56 +140,68 @@ export default Vue.extend({
...
     previousIndex: 0, //to prevent undefined error when clicking on the same institution twice
-    tempInstitutionSorted: new Array<Institution>(),
+    tempInstitutionSorted: new Array<ListItem>(),
     info: false,
     help: false,
+    validInstitutions: true,
   }),
   methods: {
-    ...mapActions(['updateInstitutions', 'resetPanels']),
+    ...mapActions(['updateInstitutions', 'resetInstitutions']),
```
Comment (cmonnom) [inline src/components/manage/BuildExploreInstitutions.vue:149]: Good catch here.

File: src/types/panel-types.ts [hunk 4]
```diff
@@ -193,6 +193,7 @@ export class ParsedGenes {
  */
 export class PanelBuilder {
   panelName!: string
+  genes!: Gene[]
```
Comment (cmonnom) [inline src/types/panel-types.ts]: I think we don't use this field anymore. You should be able to delete it. (it's not in the current branch anyway)

**Review by cmonnom — CHANGES_REQUESTED** [review]
Looks good. Mostly a couple of unused fields and a method to refactor.

File: src/types/panel-types.ts (hunk 4 above)
Comment (skriyaz95) [reply src/types/panel-types.ts]: done

File: src/components/manage/BuildExploreInstitutions.vue (hunk 3 above)
Comment (skriyaz95) [reply src/components/manage/BuildExploreInstitutions.vue:149]: done

File: src/components/home/PanelResult.vue (hunk 2 above)
Comment (skriyaz95) [reply src/components/home/PanelResult.vue:336]: done

Comment (skriyaz95) [reply src/components/home/PanelResult.vue:336]: yes I ll update this asap

File: src/components/explore/ListTemplate.vue (hunk 1 above)
Comment (skriyaz95) [reply src/components/explore/ListTemplate.vue]: done

**Review by cmonnom — APPROVED** [review]

**Pull request merged**